File: frameworks/PHP/sw-fw-less/config/router.php
<?php

use App\services\TestService;

return [
    'single' => [
        ['GET', '/json', [TestService::class, 'json']],
        ['GET', '/db', [TestService::class, 'db']],
        ['GET', '/queries', [TestService::class, 'queries']],
        ['GET', '/fortunes', [TestService::class, 'fortunes']],
        ['GET', '/updates', [TestService::class, 'updates']],
        ['GET', '/plaintext', [TestService::class, 'plaintext']],
    ],
    'group' => [],
];

File: frameworks/Elixir/phoenix/lib/hello_web/router.ex
defmodule HelloWeb.Router do
  use HelloWeb, :router

  pipeline :browser do
    plug :accepts, ~w(html json)
  end

  scope "/", HelloWeb do
    pipe_through [:browser]

    get "/json", PageController, :_json
    get "/db", PageController, :db
    get "/queries", PageController, :queries
    get "/fortunes", PageController, :fortunes
    get "/updates", PageController, :updates
    get "/plaintext", PageController, :plaintext
    get "/cached-queries", PageController, :cached
    get "/", PageController, :index
  end
end
